src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -3417,8 +3432,42 @@ components:
         dataProject:
           type: string
           description: Project id of the snapshot data project
+        accessInformation:
+          $ref: '#/components/schemas/SnapshotAccessInfoModel'
+          description: How to access the metadata for the snapshot
       description: >
         SnapshotModel returns detailed data about an existing snapshot.
+    SnapshotAccessInfoModel:
+      type: object
+      properties:
+        bigQuery:
+          $ref: '#/components/schemas/SnapshotAccessInfoModelBigQuery'
+    SnapshotAccessInfoModelBigQuery:
+      type: object
+      properties:
```

Comment:
Would it be helpful to add descriptions for any of these new properties?

Reply:
Yeah, that's a good idea

Reply:
Also, I realized that my artisanal hand crafted open api code is using a deprecated method of requiring fields so will fix that